src/main/java/core/basesyntax/service/impl/BalanceAction.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.service.ActionHandler;
+
+public class BalanceAction implements ActionHandler {
```

Comment:
Please follow the encapsulation principle. The method `apply` should have an access modifier. In Java, it's recommended to explicitly define the access level, even if it's the default one.